src/MockClassifier.UnitTests/Services/Dmr/DmrServiceTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Microsoft.Extensions.Logging;
+using MockClassifier.Api.Services.Dmr;
+using MockClassifier.UnitTests.Extensions;
+using Moq;
+using RichardSzalay.MockHttp;
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace MockClassifier.UnitTests.Services.Dmr
+{
+    public class DmrServiceTests
+    {
+        private static readonly DmrServiceSettings DefaultServiceConfig = new DmrServiceSettings
+        {
+            DmrApiUri = "https://dmr.fakeurl.com"
+        };
+
+        private readonly Mock<IHttpClientFactory> httpClientFactory = new Mock<IHttpClientFactory>();
+        private readonly MockHttpMessageHandler httpMessageHandler = new MockHttpMessageHandler();
+        private readonly Mock<ILogger<DmrService>> logger = new Mock<ILogger<DmrService>>();
+
+        [Fact]
+        public async Task ShouldCallDmrApiWithGivenRequest_WhenRequestIsRecorded()
+        {
+            httpMessageHandler.SetupWithMessage();
+
+            var clientFactory = GetHttpClientFactory(httpMessageHandler);
+
+            var sut = new DmrService(clientFactory.Object, DefaultServiceConfig, logger.Object);
```

Comment:
`sut` is setup in each test. Could it be setup once in the constructor instead?

Reply:
Unfortunately, that won't be possible since the result of the `HttpClientFactory` changes between tests and I can only set that up in each test :(